CHANGES.rst

```diff
@@ -1,21 +1,27 @@
 Changes
 =======
 
-Version 0.8.1 (UNRELEASED)
+Version 0.8.1 (2022-02-09)
 --------------------------
 
 - Users:
     - Adds support for specifying ``kubernetes_job_timeout`` for Kubernetes compute backend jobs.
+    - Fixes workflow stuck in pending status due to early Yadage failures.
```

Comment:
Perhaps we should also mention [this fix](https://github.com/reanahub/reana-server/pull/438/files) to solve Kubernetes job memory limits issue which resulted in workflows stuck in a queued state for users